rts/motoko-rts/src/gc/mark_compact.rs

```diff
@@ -112,6 +116,13 @@ unsafe fn mark_compact<M: Memory, SetHp: Fn(u32)>(
         thread(continuation_table_ptr_loc);
     }
 
+    if (*region0_ptr_loc).is_ptr() {
+        mark_object(mem, *region0_ptr_loc);
+        // Similar to `mark_root_mutbox_fields`, `region0_ptr_loc` is in static heap so
+        // it will be readable when we unthread the continuation table
```

Comment:
```suggestion
        // it will be readable when we unthread the continuation table
```
Does this comment need fixing? Looks like cut-n-paste from above